src/hooks/index.ts

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useMemo, useState } from "react";
+import initSdk, { SafeInfo, SdkInstance } from "@gnosis.pm/safe-apps-sdk";
+
+import provider from "../config/provider";
+
+export function useAppsSdk(): [SdkInstance, SafeInfo | undefined] {
+  const [safeInfo, setSafeInfo] = useState<SafeInfo>();
+
+  const safeMultisigUrls: RegExp[] = useMemo(() => {
+    const urls: RegExp[] = [];
+    urls.push(/http:\/\/localhost:3000/);
+    return urls;
+  }, []);
+  const [appsSdk] = useState<SdkInstance>(initSdk(safeMultisigUrls));
+
+  /* For development purposes with local provider */
+  useEffect(() => {
+    if (process.env.NODE_ENV === "development") {
```

Comment:
I'm not sure this is the best way to do this. This prevents us from testing transactions being sent using the gnosis ui without doing a production build.

It might be best to have a double lock on this? If we use `process.env.NODE_ENV !== "production" && REACT_APP_LOCAL_WEB3_PROVIDER` then we have the choice of using a local provider in testing but ensure that it never gets set incorrectly in production.